test/storage_init_test.go

```diff
@@ -338,9 +340,215 @@ var _ = Describe(
 				)
 			},
 		)
+
+		Context(
+			"When doing PVC change", func() {
+
+				clusterNamespacedName := getNamespacedName(
+					clusterName, namespace,
+				)
+
+				BeforeEach(
+					func() {
+						zones, err := getZones(ctx, k8sClient)
+						Expect(err).ToNot(HaveOccurred())
+
+						zone1 := zones[0]
+						zone2 := zones[0]
+						racks := []asdbv1.Rack{
+							{ID: 1, Zone: zone1},
+							{ID: 2, Zone: zone2},
+						}
```

Comment:
If we are deploying in the same zone, then I think there is no need to set the zone, rack id is sufficient